frontend/src/components/common/form/runtimeConfigurationInput.jsx

```diff
@@ -184,23 +189,17 @@ export function EditRuntimeConfiguration(props) {
       <ContentSection
         className="bg-darker"
         id="edit_runtime_configuration-section"
-        style={{ width: "45%", maxHeight: "590px", overflowY: "auto" }}
+        style={{ width: "45%", height: "590px", overflowY: "auto" }}
       >
         <small className="text-muted">
           Note: Edit this only if you know what you are doing!
         </small>
-        <CustomJsonInput
-          id="edit_runtime_configuration-modal"
-          placeholder={editableConfig}
+        <JsonEditor
+          id="runtime_configuration"
+          initialJsonData={editableConfig}
           onChange={setJsonInput}
-          /* waitAfterKeyPress=1000 is the default value and we cannot change it:
-              with this value (or higher) in case the user press "save & close" too fast it doesn't take changes.
-              If we decrease it (min allowed 100) we don't have this problems, but it's not possible to edit:
-              The library auto refresh and move the cursor too fast to make it editable.
-            */
-          waitAfterKeyPress={1000}
-          height="590px"
-          width="450px"
+          height="540px"
+          width="430px"
```

Comment:
I agree with px for height (it's scrollable), but maybe % is better for width, what do you think ?